inc/container.class.php

```diff
@@ -1587,13 +1587,6 @@ public static function getAddSearchOptions($itemtype, $containers_id = false)
 
         $i = 76665;
 
-        $search_string = json_encode($itemtype);
-        // Backslashes must be doubled in LIKE clause, according to MySQL documentation:
-        // > To search for \, specify it as \\\\; this is because the backslashes are stripped
-        // > once by the parser and again when the pattern match is made,
-        // > leaving a single backslash to be matched against.
-        $search_string = str_replace('\\', '\\\\', $search_string);
-
```

Comment:
This should not be reverted. It is mandatory to correctly handle itemtype using namespaces (i.e. `Glpi\Socket`).